spec/doc.md

```diff
@@ -120,22 +120,22 @@ To resume **from** valid parameters are listed below. A parameter can be optiona
 
 | name | type | mandatory | prefix | Complementary information |
 |------|------|-----------|--------|---------------------------|
-| from | Integer,String | <i class="fas fa-check"></i> | from | Only on Warp 10, [ISO 8601 dates string](https://en.wikipedia.org/wiki/ISO_8601) |
-| from | Integer, Double, String |<i class="fas fa-check"></i>| from | Only on Prometheus, [RFC3339 format](https://www.ietf.org/rfc/rfc3339.txt) |
-| to | Integer,String | <i class="fas fa-times"></i> | to | Only on Warp 10, [ISO 8601 dates string](https://en.wikipedia.org/wiki/ISO_8601) |
-| to | Integer, Double, String |<i class="fas fa-times"></i> | to | Only on Prometheus, [RFC3339 format](https://www.ietf.org/rfc/rfc3339.txt) |
+| from | Integer,String | yes | from | Only on Warp 10, [ISO 8601 dates string](https://en.wikipedia.org/wiki/ISO_8601) |
```

Comment:
You can use `[]` and `[x]`
https://help.github.com/en/github/writing-on-github/basic-writing-and-formatting-syntax#task-lists